### PR TITLE
Fixed error message, removed mention of ghost --refresh

### PR DESCRIPTION
**Pull request by vritant**

### CODE REVIEW

File: src/subscription_manager/managercli.py [hunk 1]
```diff
@@ -995,7 +995,7 @@ def _validate_options(self):
         elif (self.options.activation_keys and not self.options.org):
             system_exit(os.EX_USAGE, _("Error: Must provide --org with activation keys."))
         elif (self.options.force and self.options.consumerid):
-            system_exit(os.EX_USAGE, _("Error: Can not force registration while attempting to recover registration with consumerid. Please use --force without --consumerid to re-register or --refresh"))
+            system_exit(os.EX_USAGE, _("Error: Can not force registration while attempting to recover registration with consumerid. Please use --force without --consumerid to re-register or use the clean command and try again without --force"))
```
Comment (alikins) [inline src/subscription_manager/managercli.py]: Needs a period at the end.

Comment (vritant) [inline src/subscription_manager/managercli.py]: @alikins done

**Comment by Ceiu** [comment]
retest this please

**Comment by alikins** [comment]
ACK.

**Pull request merged**